website/citations/providers.py

```diff
@@ -105,7 +105,7 @@ def user_accounts(self, user):
                 self.serializer(
                     user_settings=user.get_addon(self.provider_name) if user else None
                 ).serialize_account(each)
-                for each in user.external_accounts
+                for each in user.external_accounts.all()
```

Comment:
@erinspace I probably led you astray with the `.all()`. You could probably use a `.filter(provider=provider_name)` to avoid the python loops. It'll work for now. If you want to fix it and test you're a rockstar. If you want to add a TODO and leave it for the optimization pass that's okay too. Sorry for the inadequate advice.